io.github.Q1CHENL.fig.json

```diff
@@ -0,0 +1,112 @@
+{
+  "id": "io.github.Q1CHENL.fig",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "fig",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=xdg-run/gvfsd",
+    "--filesystem=home",
+    "--filesystem=xdg-pictures:rw",
+    "--filesystem=xdg-download:rw",
+    "--env=GDK_BACKEND=x11",
```

Comment:
this is incompatible with the socket permissions above because it disable wayland.